### PR TITLE
X0129 load studies from mlwh

### PR DESCRIPTION
**Pull request by sabrine33**

### CODE REVIEW

**Review by seenanair — COMMENTED** [review]
I couldn't make it work with server running even after updating  application-dev-properties with the fields you mentioned. It might be something to do with my settings, but it is worth checking at your end to make sure everything is working as intended.

File: src/components/workAllocation/WorkAllocation.tsx [hunk 1]
```diff
@@ -379,13 +390,12 @@ export default function WorkAllocation() {
...
-                  label={'Number of original samples'}
-                  name={'numOriginalSamples'}
-                  type={'number'}
-                  maxLength={MAX_NUM_BLOCKANDSLIDES}
-                  min={0}
-                />
+                <FormikInput type={'number'} label="DNAP study ID" name="ssStudyId" />
+                {studyName && (
+                  <div className={'flex-row whitespace-nowrap space-x-2 p-0'}>
+                    {studyName && <Pill color={'blue'}>{studyName}</Pill>}
```
[the remark this answers is not in the document]
Comment (sabrine33) [reply src/components/workAllocation/WorkAllocation.tsx:396]: @seenanair in the work number, it appears in blue and pink based on certain conditions. I'm not sure where else the Pill is used, but I opted for blue as it is a "safe" color. Red/pink should have a specific color code interpretation, and that's my perspective. However, I'm open to changes.

[the remark this answers is not in the document]
Comment (seenanair) [reply src/components/workAllocation/WorkAllocation.tsx:396]: Ok, if that is used already you don't need to probably change it. In STAN, pink is used to make information stand out , used in buttons as well I think.

**Review by seenanair — COMMENTED** [review]
Looking good, just two minor suggestions.

**Review by seenanair — APPROVED** [review]
Thank you for the update. Looking good 👍

**Comment by sabrine33** [comment]
merge to devel

**Pull request merged**